frontend_lib/build-script.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
```

Comment:
is such a bash file mandatory ?
I would expect 2 different command lines, doing their own magic, for each bundle type

Couldn't all this intelligence be added in respective scripts ?

It is allowed to have yarn commands to execute node scripts instead of bash scripts

Having bash script at the root of Tracim make more sense since there is the backend part.
Here, in frontend_lib, we are in a fully javascript folder, a node script would be more intuitive I think

Reply:
I want them in the same script to avoid duplicating code.
This script is a deduplication of what what becoming long lines in package.json's build commands. Using a shell script was straightforward since this was the language used in the build script but I can write it in javascript if you think it's worth the effort.

Reply:
We will open an issue to think about bash scripts to build the frontend(and tracim in general)